### ISSUE TITLE
Failed to verify bitcode in CobrowseIO.framework/CobrowseIO:

### ISSUE THREAD

**Issue by qadirsuh** [cobrowseio/cobrowse-sdk-ios-binary#2]
I am trying to build and exportArchive my app with [fastlane](https://fastlane.tools/).

I am getting error following error.

```
xcodebuild -exportArchive -exportOptionsPlist /var/folders/1r/bl8hkl410fqdwdhsv_2sqy6c0000gn/T/gym_config20181026-31677-15gsj71.plist -archivePath '/Users/qadir/Library/Developer/Xcode/Archives/2018-10-26/jamesAppV2 2018-10-26 16.43.46.xcarchive' -exportPath /var/folders/1r/bl8hkl410fqdwdhsv_2sqy6c0000gn/T/gym_output20181026-31677-mtv0k3
2018-10-26 16:52:42.376 xcodebuild[38718:190029] [MT] IDEDistribution: -[IDEDistributionLogging _createLoggingBundleAtPath:]: Created bundle at path '/var/folders/1r/bl8hkl410fqdwdhsv_2sqy6c0000gn/T/jamesAppV2_2018-10-26_16-52-42.375.xcdistributionlogs'.
error: exportArchive: Failed to verify bitcode in CobrowseIO.framework/CobrowseIO:
error: Bundle only contains bitcode-marker /var/folders/1r/bl8hkl410fqdwdhsv_2sqy6c0000gn/T/XcodeDistPipeline.1H6/Root/Payload/jamesAppV2.app/Frameworks/CobrowseIO.framework/CobrowseIO (arm64)

Error Domain=IDEFoundationErrorDomain Code=1 "Failed to verify bitcode in CobrowseIO.framework/CobrowseIO:
error: Bundle only contains bitcode-marker /var/folders/1r/bl8hkl410fqdwdhsv_2sqy6c0000gn/T/XcodeDistPipeline.1H6/Root/Payload/jamesAppV2.app/Frameworks/CobrowseIO.framework/CobrowseIO (arm64)

" UserInfo={NSLocalizedDescription=Failed to verify bitcode in CobrowseIO.framework/CobrowseIO:
error: Bundle only contains bitcode-marker /var/folders/1r/bl8hkl410fqdwdhsv_2sqy6c0000gn/T/XcodeDistPipeline.1H6/Root/Payload/jamesAppV2.app/Frameworks/CobrowseIO.framework/CobrowseIO (arm64)

}

** EXPORT FAILED **
[16:54:30]: Exit status: 70
```

What could be the issue? any clue.

**Comment by headlessme** [comment]
Looks like this could be a regression in our release process to do with Xcode 10. Could you try the version we just released (version 1.6.2) and see if that has resolved the issue?

**Comment by qadirsuh** [comment]
Thanks for this.
yes it resolved my issue. no more `Failed to verify bitcode in CobrowseIO.framework/CobrowseIO` error.
